src/Explorer/index.jsx

```diff
@@ -470,6 +565,12 @@ class Explorer extends React.Component {
             }
           </div>
           <div className='explorer__visualizations'>
+            <Button
+              className='explorer-visualization__download-button'
```

Comment:
this button looks different from Explorer buttons on data-portal - i think it would be better to have the same style in both
not hover should have a border
hover should be brighter orange

Reply:
oh looks like it's all orange buttons in NDE portal. so i guess it's not in the scope of your ticket... but still i think it would be better

Reply:
yeah the NDE portal has some CSS in `src/Index/page.less` that overrides the button style.

I don't think there is a particular reason they are in there though

Reply:
🤔  seems all buttons in ecosystem portal is different from normal portal. I am still looking at it. Will submit another PR to fix that.